lib/inspec/profile.rb

```diff
@@ -808,9 +808,11 @@ def archive(opts)
       # TODO ignore all .files, but add the files to debug output
 
       # Generate temporary inspec.json for archive
-      if opts[:export]
+      export_opt_enabled = opts[:export] || opts[:legacy_export]
+      if export_opt_enabled
+        info_for_profile_summary = opts[:export] ? info_from_parse : info
```

Comment:
This is wrong - the conditional should be:
```
opts[:legacy_export] ? info : info_from_parse
```

Reply:
I believe it is not wrong.

https://github.com/inspec/inspec/blob/532e573671ccc1c57a83f2201932cdee2705f9fb/lib/inspec/profile.rb#L813


The above line is checked only if `export_opt_enabled` is true i.e. either the flag `--export` or `--legacy-export` is passed in the cli and is part of the `opts`.

This means the conditions - 
```
opts[:export] ? info_from_parse : info
```
and 
```
opts[:legacy_export] ? info : info_from_parse
```
are both going to equivalent.

Reply:
Let me know if you mean something else here, which I maybe missing

Reply:
Addressed!